data.json

```
@@ -131,7 +131,7 @@
           "name": "GoosePresence",
           "name-internal": "goosepresence",
           "resourcehub": "https://desktopgooseunofficial.github.io/ResourceHub/mods/GoosePresence.html",
-          "url": "https://github.com/VisualError/GoosePresence/releases/download/1.0/GoosePresence.zip"
+          "url": "https://github.com/VisualError/GoosePresence/releases/latest/download/GoosePresence.zip"
```

```suggestion
          "url": "https://github.com/VisualError/GoosePresence/releases/download/1.0.1/GoosePresence.zip"
```
you usually want to refrain from using latest because that means the mod version in the backend may be out of date (change mod-version to 1.0.1 as well if this is accepted)